Gordon360/Services/RecIM/SeriesService.cs

```diff
@@ -394,9 +395,30 @@ private async Task<IEnumerable<MatchViewModel>> ScheduleRoundRobin(int seriesID,
                     var teamIDs = new List<int>() { teams[i], teams[j] };
                     if (!teamIDs.Contains(0))
                     {
+                        DateTime matchStartTime;
+                        try
+                        {
+                            matchStartTime = day.ConvertToUtc(Time_Zones.EST);
+                        }
+                        catch
+                        {
+                            // edgecase where a match is scheduled ON daylight saving
+                            day = day.AddMinutes(60 + schedule.EstMatchTime);
```

Comment:
1. I believe that you don't need to check for Daylight Savings when converting **TO** UTC - UTC time will always be valid because it doesn't observe Daylight Savings, so this will never fail.
2. You should put this logic for handling Daylight Savings into the `ConvertFromUtc` method so that it's handled consistently and not repeated sporadically.
3. This only handles the `Spring Forward` case of Daylight Savings, it doesn't handle the `Fall Back` case.

Reply:
Handled by reimplementing the inverse of current datetime logic to handle all calculations back in UTC with Mon-Sun checked via conversion to EST (thank you @EjPlatzer for the suggestion)